server/src/main/java/org/elasticsearch/script/ScriptStats.java

```diff
@@ -75,7 +87,11 @@ public ScriptStats(ScriptContextStats context) {
         );
     }
 
-    public ScriptStats(StreamInput in) throws IOException {
+    public static ScriptStats of(StreamInput in) throws IOException {
```

Comment:
I would prefer to name this `read` to be more inline with the rest of the code base.